clang/include/clang/Basic/Attr.td

```diff
@@ -4561,3 +4561,9 @@ def CodeAlign: StmtAttr {
     static constexpr int MaximumAlignment = 4096;
   }];
 }
+
+def ClspvLibclcBuiltin: DeclOrStmtAttr {
+  let Spellings = [Clang<"clspv_libclc_builtin">];
+  let Documentation = [Undocumented];
+  let SimpleHandler = 1;
```

Comment:
This should also probably set 'appertainment' via `Subjects`.

Reply:
Also, is there a reason as to why it’s a `DeclOrStmtAttr` instead of an `InheritableAttr`? Because afaik most of the other function attributes use the latter.

Reply:
I have never dealt with clang attribute implementation, so I have no idea whether `DeclOrStmtAttr` or `InheritableAttr` should be used. I'll change it for `InheritableAttr`.